### ISSUE TITLE
Only last private compile parameter override processed from subdir Rules.mk

### ISSUE THREAD

**Issue by sbestgen** [IBM/ibmi-bob#201]
![image (3)](https://user-images.githubusercontent.com/107139127/218523788-9f593bc2-2815-49f9-948a-24728c70e5b1.png)

**Comment by edmundreinhardt** [comment]
Problem is on this line
```typescript
                    target, variable = line.strip().split(':')
                    var_name, var_value = variable.split('=')
                    variables[target.strip()] = (var_name.strip(), var_value.strip())
```
https://vscode.dev/github/IBM/ibmi-bob/blob/6e5a562795cb7f41a4bd12e29aa50bdec2b51084/src/makei/rules_mk.py#L220-L221

**Comment by edmundreinhardt** [comment]
Fixed in Bob 2.4.11
